### ISSUE TITLE
Make it possible to search in extended using keyboard

### ISSUE THREAD

**Issue by majsan** [spraakbanken/korp-frontend#85]
Currently it is impossible to search in extended using keyboard only, since it is not possible to use tab to move to the search-button. This should be fixed.

We should also enable searching using Enter. Only when dropdown/select is open of course.

**Comment by arildm** [comment]
This has been fixed at some point.
